extensions-contrib/cassandra-storage/src/main/java/org/apache/druid/storage/cassandra/CassandraStorage.java

```diff
@@ -66,7 +66,8 @@ public CassandraStorage(CassandraDataSegmentConfig config)
     this.config = config;
     indexStorage = new CassandraChunkedStorageProvider(keyspace, INDEX_TABLE_NAME);
 
-    descriptorStorage = new ColumnFamily<String, String>(DESCRIPTOR_TABLE_NAME,
-        StringSerializer.get(), StringSerializer.get());
+    descriptorStorage = new ColumnFamily<>(DESCRIPTOR_TABLE_NAME,
```

Comment:
Please revert the formatting changes from this PR to keep it more straightforward.